index.html

```diff
@@ -285,6 +284,53 @@ <h3>Skills</h3>
           <li>Terminal</li>
         </ul>
       </section>
+      <section class="form">
+        <h3>I'm always <br> interested in hearing <br> about new projects, <br> so if you'd like to <br> chat please get in <br> touch.</h3>
+        <form action="/actio-handle">
+          <ul>
+          <li>
```

Comment:
- [ ] I would suggest you kindly add a `form action` attribute to integrate the Formspree service. Kindly check this [link](https://formspree.io)  to get a Formspree account and you can make use of the address ID generated. 😃 😃 

- [ ]  Also I would suggest you make use of a `method` attribute to have a value equal to `POST` 😃 😃 

Hint: 👇 

```HTML
<form action="https://formspree.io/f/{form_id}" method="post">
</form>
```